card/spec/lib/card/format_spec.rb

```diff
@@ -17,7 +17,7 @@
       expect(format.show_view?( :menu, optional_menu: :hide                             )).to be_falsey
     end
 
-    it "should handle args from inclusions" do
+    it "should handle args from nests" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.